src/data/faq.json

```diff
@@ -778,7 +778,17 @@
                         "textblock": [
                             "You can find more information in our <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution Architecture Document</a>."
                         ]
-                    }
+                    },
+                    {
+                        "title": "Contact Diary",
+                        "anchor": "contact_diary",
+                        "active": true,
+                        "textblock": [
+                            "Since release of the contact diary function, we continuously received suggestions for changes and improvements. All those ideas will be reviewed technically and together with the client regarding:",
+                            "<ul><li>Benefit for the consumers of the diary (e.g., the health authorities)</li><li>User friendliness of the UI,</li><li>extension by functions (e.g. recording of additional parameters),</li><li>technical feasibility (or the effort required for implementation), and</li><li>legal framework and limits</li></ul>",
```

Comment:
remove comma after e.g.

(e.g., the health authorities) => (e.g. the health authorities)

Capitalise each entry in the list to be consistent:
extension => Extension 
technical => Technical 
legal => Legal